src/main/java/ro/vladutit/Don/t/forget/v2/controller/CategoryController.java

```diff
@@ -41,6 +41,8 @@ public void deleteItem(@PathVariable (value = "id") Long id) {
     @GetMapping("/categories/{name}")
     @ResponseBody
     public boolean checkCategoryName(@PathVariable(value = "name") String name) {
-        return categoryService.getCategoryByName(name);
+        boolean res = categoryService.getCategoryByName(name);
```

Comment:
let's call it 'result' please, let's be super clear

Reply:
It was temporary for debugging only. Now I can return directly without saving in a variable.